integration-test/tests/cms/about.us.test.ts

```diff
@@ -1,25 +1,48 @@
 import { expect, test } from '@playwright/test';
 import { validateSchema } from '@utils/helpers/schema.validation';
 import { aboutSchema } from '@utils/datafactory/schemas/about.schema';
+import { aboutUsPageData } from '@utils/datafactory/test-data/about.us.page.data';
 
-test('GET /api/cms/v1/about returns correct about us data', async ({ request }) => {
-  const response = await request.get(`/api/cms/v1/about`);
+test.describe('Validate positive test cases for ABOUT Page API', () => {
+  test.beforeEach(async ({ request }) => {
```

Comment:
Could we change to be beforeAll ?

Reply:
In the future, when we have more test cases inside, it makes sense to change to `beforeAll`